src/components/delegates/delegates.js

```diff
@@ -41,11 +42,12 @@ app.component('delegates', {
       this.loading = true;
       this.usernameInput = '';
       this.usernameSeparator = '\n';
+      this.delegates = [];
 
       this.updateAll();
 
       this.$scope.$watch('search', (search, oldValue) => {
-        this.delegatesDisplayedCount = 20;
+        this.delegates = [];
```

Comment:
With this solution we're facing 2 issues:
 - with every change on search phrase we're performing the heavy task of compiling the whole lisk. as we know angular's Achilles' heel. so we end up a glitchy behaviour.
 - Some times the doesn't fill the screen and you only can scroll in a small chunk of the screen whilst the rest of it is empty.
![screen shot 2017-05-30 at 12 02 25](https://cloud.githubusercontent.com/assets/666685/26578563/a54b82b0-4530-11e7-8e08-1a511e889ebb.png)